web/satellite/src/components/account/billing/billingTabs/PaymentMethods.vue

```diff
@@ -1,22 +1,944 @@
+                    >
+                        <img
+                            v-if="isLoading"
+                            class="payment-loading-image"
+                            src="@/../static/images/account/billing/loading.gif"
+                            alt="loading gif"
+                        >
+                        <SuccessImage
+                            v-if="isLoaded"
+                            class="payment-loaded-image"
+                        />
+                        <span class="add_card_button_text">Add Credit Card</span>
+                    </div>
+                </div>
+            </div>
+        </div>
+        
+        <div v-if="isRemovePaymentMethodsModalOpen || isChangeDefaultPaymentModalOpen" class="edit_payment_method">
+            <div v-if="isChangeDefaultPaymentModalOpen" class="change-default-modal-container">
+                <CreditCardImage class="card-icon-default" />
+                <div class="edit_payment_method__container__close-cross-container-default" @click="onCloseClickDefault">
+                    <CloseCrossIcon class="close-icon" />
+                </div>
+                <div class="edit_payment_method__header">Select Default Card</div>
+                <form v-for="card in creditCards" :key="card.id"> 
+                    <div class="change-default-input-container">
+                        <AmericanExpressIcon v-if="card.brand === 'amex' " class="cardIcons" />
+                        <DiscoverIcon v-if="card.brand === 'discover' " class="cardIcons" />
+                        <JCBIcon v-if="card.brand === 'jcb' " class="cardIcons jcb-icon" />
+                        <MastercardIcon v-if="card.brand === 'mastercard' " class="cardIcons mastercard-icon" />
+                        <UnionPayIcon v-if="card.brand === 'unionpay' " class="cardIcons union-icon" />
+                        <VisaIcon v-if="card.brand === 'visa' " class="cardIcons" />
+                        <DinersIcon v-if="card.brand === 'diners' " class="cardIcons diners-icon" />
+                        <img src="@/../static/images/payments/cardStars.png" alt="Hidden card digits stars image" class="payment-methods-container__card-container__info-area__info-container__image"> 
+                        {{ card.last4 }}
+                        <input 
+                            :id="card.id" 
+                            v-model="defaultCreditCardSelection"  
+                            :value="card.id" 
+                            class="change-default-input" 
+                            type="radio" 
+                            name="defaultCreditCardSelection"
+                        >
+                    </div>
+                </form>
+                <div class="default-card-button" @click="updatePaymentMethod">
+                    Update Default Card
+                </div>
+            </div>
+            <div v-if="isRemovePaymentMethodsModalOpen" class="edit_payment_method__container">
```

Comment:
nit - the text styling and top padding of this component looks off to me when testing locally in chrome. Specifically, I notice very small top padding, and a serif font being used for "Edit default card"